### DISCUSSION TITLE
LIST | 可供社科(经管)领域使用的数据集汇总

### DISCUSSION THREAD

**Discussion by giscus** [hiDaDeng/hidadeng.github.io#41] [交流]
# LIST | 可供社科(经管)领域使用的数据集汇总

可供社科(经管)使用的数据集

https://textdata.cn/blog/datasets_available_for_management_science/

**Comment by hiDaDeng** [comment]
也可发布有偿任务， 需要什么数据， 留下联系方式， 让大家帮你找找。
